newscript.go

```diff
@@ -308,7 +332,7 @@ func dd() error {
 		}
 	}
 	fmt.Printf("Running dd to put the new kernel onto the desired location on the usb.\n")
-	args := []string{"dd", "if=newKern", "of=" + location}
+	args := []string{"dd", "if=" + *nameKernel, "of=" + location}
```

Comment:
do you use this function anymore? up above you replace it with cp? I'm confused.

Reply:
actually you should still use this function so doing the cp to the usb stick makes no sense. You should never run this program as root.